CAP/gap/MethodRecord.gi

```diff
@@ -3131,7 +3131,67 @@ SolveLinearSystemInAbCategory := rec(
   return_type := "morphism_or_fail"
 ),
 
-  ) );
+RandomObjectByInteger := rec(
```

Comment:
I am not sure if this is worth it, but do you want to specify `io_types` for those entries?